### ISSUE TITLE
QA for watcher leave stopped instance

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#561]
Originally reported of Google Code with ID 505.

```
qa_daemon.TestInstanceConsecutiveFailures() violates one assumption of QA: tests shouldn't change the cluster state. Instead, it stops an instance and leaves it stopped at the end. It should restart the instance before exiting.

It affects 2.7 too.
```
Originally added on 2013-06-25 22:06:53 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
I didn't notice that it was already fixed in 2.7 but not yet merged in master
```
Originally added on 2013-06-25 22:11:32 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2013-07-09 11:01:26 +0000 UTC.

Changed State: Released
